resources/deployments/controller/controller_deployments.go

```diff
@@ -65,7 +65,7 @@ func (d *DeploymentsController) PostDeployment(w rest.ResponseWriter, r *rest.Re
 		return
 	}
 
-	d.view.RenderSuccessPost(w, r, id)
+	d.view.RenderSuccessPost(w, r, "./deployments/"+id)
```

Comment:
My point was that controller also won't know anything specific about routing. This way we add yet another place that needs to be tweaked when messing around with route naming.

`w.Header().Add(HttpHeaderLocation, fmt.Sprintf("./%s/%s", strings.TrimLeft(r.URL.Path, "/api/0.0.1/), id))`

This way this will be universal and work for paths (outside of tweaks to common route).

When we align routing with gateway we can then simplify to `r.URL.Path/id`, no changes outside of module required.

What do you think ?